application.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+import click
+
+
+@click.command()
```

Comment:
their should be command name like search

Reply:
Done with the change.